### PR TITLE
Wiring in of task details macro system

### PR DESCRIPTION
**Pull request by toddtarsi**
This pretty much does 0 UX for the page, but lays the groundwork for adding and removing the task details page while re-using a macro with SSR

### CODE REVIEW

File: theme/p2/taskinfo_details.p2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% macro taskinfo_details(task) export %}
+  <div>
+    Foo
+  </div>
+{% endmacro %}
+
+{{taskinfo_details(task)}}
```
Comment (the-maldridge) [inline theme/p2/taskinfo_details.p2:7]: Missing newline at the end of this file.

**Review by the-maldridge — CHANGES_REQUESTED** [review]

File: internal/http/taskinfo/taskinfo.go [hunk 2]
```diff
@@ -25,6 +26,33 @@ type TaskInfo struct {
 	tmpls *pongo2.TemplateSet
 }
 
+func init() {
```
Comment (the-maldridge) [inline internal/http/taskinfo/taskinfo.go:29]: I would prefer if these filters were relocated to a pongo_filters.go file.

**Review by the-maldridge — CHANGES_REQUESTED** [review]
Overall looks great.  I think there's one major change I'd like to see style wise with how the results are visualized for trivy.  With trivy now showing the vulnerabilities grouped by target, it would be nice to resurface this by target:

In the tile at the top, show each target and the count for that target specifically.
In the detail, show a distinct table for each target that was scanned, with the target metadata.

If at all possible, it would also be nice to tweak the column widths to keep the CVE and More Info links as one line items.

File: internal/http/filters/supplemental.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package supplemental
```
Comment (the-maldridge) [inline internal/http/filters/supplemental.go:1]: this should be `package filters`.

File: cmd/yurt/main.go [hunk 4]
```diff
@@ -1,7 +1,9 @@
 package main
 
 import (
+	// additional custom filters
```
Comment (the-maldridge) [inline cmd/yurt/main.go:4]: This comment should appear directly above the blank import.

File: theme/static/js/taskInfo.js [hunk 5]
```diff
@@ -1,27 +1,62 @@
...
   };
 
-  const removeClass = (className) => (el) => {
-    el.classList.remove(className);
+  const resetActiveClasses = () =>
+    individualTaskElements.forEach((el) => el.classList.remove("active"));
+
+  /*
+   * This handler does a few things:
+   * 1. Makes the root grid work as a
```
Comment (the-maldridge) [inline theme/static/js/taskInfo.js:18]: This comment looks incomplete?

**Review by the-maldridge — CHANGES_REQUESTED** [review]
2 nits, otherwise good.

File: internal/http/filters/supplemental.go (hunk 3 above)
Comment (toddtarsi) [reply internal/http/filters/supplemental.go]: ✅

File: cmd/yurt/main.go (hunk 4 above)
Comment (toddtarsi) [reply cmd/yurt/main.go]: ✅

File: theme/static/js/taskInfo.js (hunk 5 above)
Comment (toddtarsi) [reply theme/static/js/taskInfo.js]: ✅

**Review by the-maldridge — APPROVED** [review]

**Pull request merged**